pkg/storage/export/export/vm-source.go

```diff
@@ -146,9 +146,9 @@ func (ctrl *VMExportController) isSourceAvailableVM(vmExport *exportv1.VirtualMa
 		return false, "", err
 	}
 	if exists {
-		return false, fmt.Sprintf("Virtual Machine %s/%s is running", vmi.Namespace, vmi.Name), nil
+		return exists, fmt.Sprintf("Virtual Machine %s/%s is running", vmi.Namespace, vmi.Name), nil
```

Comment:
why return true here?

Reply:
I should probably rename that function to isSourceInUse since that is what it is doing.